metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/SquidIntegrationTest.java

```diff
@@ -29,6 +29,11 @@ String getSensorType() {
   }
 
   @Override
```

Comment:
I have a couple of things that come to mind, not all relevant:

* With 777 and 942, squid is a real parser extension, installed and usable etc.  So throwing in a test of something outside the parser into it's integration test, and making it _diffferent_ than the others seems not right to me.

* Same for any parser really.  The parsers tests are for testing the parsers.  You guys did a lot of work making the tests alike, and understandable, we should maintain that.

* If you don't change that, I don't think I'll have to do conflict work in 777, which I continue to have to maintain ;) ( disregard this )